tests/nnstreamer_filter_snpe/unittest_filter_snpe.cc

```diff
@@ -206,6 +208,55 @@ TEST (nnstreamerFilterSnpe, invoke00)
   g_free (model_file);
 }
 
+/**
+ * @brief Test snpe subplugin with successful invoke for sample dlc model (input data type: uint8)
+ */
+TEST (nnstreamerFilterSnpe, invoke01)
+{
+  int ret;
+  void *data = NULL;
+  GstTensorMemory input, output;
+  gchar *model_file;
+  GstTensorFilterProperties prop;
+
+  ASSERT_TRUE (_GetModelFilePath (&model_file, FALSE));
+  const gchar *model_files[] = {
+    model_file,
+    NULL,
+  };
+
+  const GstTensorFilterFramework *sp = nnstreamer_filter_find ("snpe");
+  ASSERT_TRUE (sp != nullptr);
+  _SetFilterProp (&prop, "snpe", model_files, FALSE);
+
+  output.size = input.size = sizeof (uint8_t) * 1;
+  input.data = g_malloc0 (input.size);
+  output.data = g_malloc0 (output.size);
+
+  ret = sp->open (&prop, &data);
+  EXPECT_EQ (ret, 0);
```

Comment:
If the failure of ```sp->open ( )``` is fatal to continue the lines after, it should be replaced with 
```suggestion
  ASSERT_EQ (ret, 0);
```

Reply:
Thanks, I updated the line.